### DISCUSSION TITLE
Faiss index lookup and Vector db lookup are deprecated

### DISCUSSION THREAD

**Discussion by vsantele** [microsoft/promptflow#2204] [Q&A] [answered]
Hi,

I just discovered that  Faiss index lookup and Vector DB lookup are marked as deprecated in VS Code.
Is there any new information, documentation, or updates on that? 

![image](https://github.com/microsoft/promptflow/assets/26800140/8a1488a2-5b2f-4ace-a634-1aef614bfdc6)

**Comment by 0mza987** [comment]
Add @Adarsh-Ramanathan  @dans-msft  to help take a look.

**Reply by dans-msft** [reply, marked as answer]
There is a replacement tool.  It is called simply "Index Lookup" and its functionality encompasses both faiss_index_lookup and vector_db_lookup (and more).  It is documented here: https://learn.microsoft.com/en-us/azure/machine-learning/prompt-flow/tools-reference/index-lookup-tool?view=azureml-api-2

It is also in the promptflow_vectordb package.

**Reply by vsantele** [reply]
Thank you,

Is "Index Lookup" shipped with promptflow_vectordb 0.2.4? I can't see it when I run `pf tool list`. 
I have promptflow 1.6.0 and promptflow-tools 1.3.0.

**Reply by Adarsh-Ramanathan** [reply]
@vsantele, you'll want to install promptflow-vectordb's `azure` extra - `pip install promptflow-vectordb[azure]`. If that doesn't work, can you share the output of running `pf tool list`?

**Reply by vsantele** [reply]
It's work with extra [azure], thank you.
